File: web-api/src/main/java/cn/labzen/web/api/annotation/Threshold.java
package cn.labzen.web.api.annotation;

import java.lang.annotation.*;

/**
 * API 并发量阈值注解。
 * <p>
 * 注解在 Controller 接口方法上，用于调整接口的并发访问量域值相关参数。
 * 可配置限流阈值、熔断策略等。
 * <p>
 * <b>待实现：</b> 定义限流参数、熔断策略等配置。
 */
@Documented
@Retention(RetentionPolicy.SOURCE)
@Target(ElementType.METHOD)
public @interface Threshold {
}

File: src/main/kotlin/cn/labzen/web/annotation/Threshold.kt
package cn.labzen.web.annotation

/**
 * 注解在 Controller 接口方法上，调整接口的并发量域值相关参数
 *
 * todo 待实现
 */
@MustBeDocumented
@Target(AnnotationTarget.FUNCTION)
@Retention(AnnotationRetention.SOURCE)
annotation class Threshold()
